### ISSUE TITLE
Subscript in equations errors as incorrect attribute reference

### ISSUE THREAD

**Issue by danyill** [asciidoctor/asciidoctor-intellij-plugin#1409]
## Observed vs. expected behavior

The plugin generates errors for subscripts in math equations.

## Steps to reproduce

Use a `stem` macro with subscripts:

For instance:

```adoc
stem:[sum(V_{drop})=sum(V_{bat})+sum(V_{sc})]

stem:[sum(V_{drop})=18.359 \text{mV}]

stem:[R=(sum(V_{drop}))/I=(18.359*10^-3)/0.235=0.078123 \text{Ohms}]

Then at nominal battery voltage of 135.32 V, maximum short circuit current cannot be more than:

stem:[I_{sc}=V/R=135.32/0.078123=1732 \text{A}]
```

This shows errors as shown in the screenshot below:

![image](https://github.com/asciidoctor/asciidoctor-intellij-plugin/assets/674783/84d7a22c-2664-490b-8108-475f7e267fc3)

## Environment

Plugin Version: 0.38.20

IntelliJ Details: 

IntelliJ IDEA 2022.1.1 (Community Edition)
Build #IC-221.5591.52, built on May 11, 2022
Runtime version: 11.0.14.1+1-b2043.45 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 10 10.0
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 12
Non-Bundled Plugins:
    com.github.bric3.excalidraw (0.3.2)
    org.asciidoctor.intellij.asciidoc (0.38.20)

Kotlin: 221-1.6.21-release-337-IJ5591.52

**Comment by danyill** [comment]
This is a beautiful issue template and a wonderful plugin! Thanks!

**Comment by ahus1** [comment]
Thank you for letting me know about this bug. A fix is available in the next release of the plugin. I'll update this issue once a pre-release is available.

**Comment by ahus1** [comment]
The change is included in pre-release 0.39.1. Please let me know if this works for you.

The pre-release of the plugin is available from [GitHub releases](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/user/installing-a-preview-release.html) and the [IntelliJ AsciiDoc EAP repository](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/user/add-eap-repository-to-ide.html).
